core/block.go

```diff
@@ -533,7 +533,7 @@ func (block *Block) Seal() error {
 }
 
 func (block *Block) String() string {
-	return fmt.Sprintf("{\"height\":%d, \"hash\":\"%s\", \"parentHash\":\"%s\", \"state\": %s, \"nonce\":%d, \"timestamp\": %d, \"coinbase\": \"%s\", \"tx\": %d}",
+	return fmt.Sprintf("{\"height\":%d, \"hash\":\"%s\", \"parentHash\":\"%s\", \"state\": \"%s\", \"nonce\":%d, \"timestamp\": %d, \"coinbase\": \"%s\", \"tx\": %d}",
```

Comment:
https://golang.org/pkg/fmt/
`%q	a double-quoted string safely escaped with Go syntax`
`\"%s\"` can be replaced by just `%q`